convert.py

```diff
@@ -9,40 +9,42 @@
 #
 
 import sys
-import json
-
-##code planning
-
-#files needed
-inputFile = ""
-outputFile = ""
+import os.path
 
 #argument bool list goes here
-verbose = False
+#argArray[0] is verbose
 
-#accept arguments
-if (len(sys.argv) == 2): # no argument, no specified output
-	inputFile = sys.argv[1]
-elif (len(sys.argv) == 3): # either with argument or specified output, not both
-	if (sys.argv[1] == "-v"): # yes verbose, no specified output
-		verbose = True
-		inputFile = sys.argv[2]
-	else: # no argument, yes specified output
-		inputFile = sys.argv[1]
-		outputFile = sys.argv[2]
-elif (len(sys.argv) == 4): # expected ONE argument AND specified output	
-	if (sys.argv[1] == "-v"): # yes verbose
-		verbose = True
-	inputFile = sys.argv[2]
-	outputFile = sys.argv[3]
-else: # no correct input syntax
-	print "USAGE: \n> ./convert.py <-arg> <input>.nvm\n> ./convert.py <-arg> <input>.nvm <output>.json"
-	input()
-	sys.exit(0)
+from arguments import *
+argArray = parseArguments()
 
+inputFile = ""
+outputFile = ""
+while True:
+	inputFile = raw_input("What file would you like to read from?\n--> ")
+	if (not os.path.isfile(inputFile)):
+		print "ERROR: File does not exist."
+	else: break
+while True:
+	outputFile = raw_input("What file would you like to write to?\n--> ")
+	if (not (outputFile.endswith('.nvm') or outputFile.endswith('.json')) ):
```

Comment:
what if the file exists? will you overwrite it?

might want to prompt for if the file exists, "do you want to overwrite" type thing